### ISSUE TITLE
Mention how to build the documentation in the README.rst

### ISSUE THREAD

**Issue by 1kastner** [brandon-rhodes/pyephem#143]
Hello there,

is it possible to mention how to build the documentation in the README.rst? That'd be a great help!

Best wishes,
Marvin

**Comment by brandon-rhodes** [comment]
Yes! It might be a few months before I get around to it, as my top priority is my more recent Skyfield library, but let's keep this issue open for the next time I return to PyEphem.

**Comment by 1kastner** [comment]
alright!

**Comment by 1kastner** [comment]
See #144
